design/FY2017/NFP-GlazingEnhancements.md

```diff
@@ -1236,7 +1236,7 @@ insert text
 
 I have got extensive help from Simon, LBNL, to develop this design document. Sincere thanks is expressed here.
 
-This section describes code changes and additions in EnergyPlus in HeatBalanceManager, CurveManager, DataHeatBalance, WindowManagerExteriorData, WindowManagerExteriorOptical, and WCE. The WCE contains 3 main projects: SPectralAveraging, SingleLayerOptics, and MultipleLayerOptics.
+This section describes code changes and additions in EnergyPlus in HeatBalanceManager, Curve, DataHeatBalance, WindowManagerExteriorData, WindowManagerExteriorOptical, and WCE. The WCE contains 3 main projects: SPectralAveraging, SingleLayerOptics, and MultipleLayerOptics.
```

Comment:
A very thorough search/replace going back 5 years or more. Not that is really matters, but if you make any other changes, I'd revert these old NFPs.